packages/app-store/zoho-bigin/api/callback.ts

```diff
@@ -0,0 +1,69 @@
+import axios from "axios";
+import type { NextApiRequest, NextApiResponse } from "next";
+import qs from "qs";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import { getSafeRedirectUrl } from "@calcom/lib/getSafeRedirectUrl";
+import prisma from "@calcom/prisma";
+
+import { decodeOAuthState } from "../../_utils/decodeOAuthState";
+import getAppKeysFromSlug from "../../_utils/getAppKeysFromSlug";
+import getInstalledAppPath from "../../_utils/getInstalledAppPath";
+
+const GRANT_TYPE = "authorization_code";
+const TYPE = "zoho-bigin_other_calendar";
+const SLUG = "zoho-bigin";
```

Comment:
Let's read all these values from config.json instead of hardcoding @ShaneMaglangit